ci_tools/bot_comment_react.py

```diff
@@ -27,9 +27,13 @@ def get_unique_test_list(keys):
     if 'pr_tests' in tests:
         tests.update(pr_test_keys)
     tests.discard('pr_tests')
+    result = list(tests)
     if 'coverage' in tests:
-        tests.add('linux')
-    return tests
+        result.append('linux')
```

Comment:
We should be careful here when using a list. If `linux` is already in the list you will end up with two identical entries.

Reply:
Oops

Reply:
Fixed b6d3319b